src/js/add_input_icon.js

```diff
@@ -216,6 +224,9 @@ async function addRelayIconToInput(emailInput) {
     const generateAliasBtn = createElementWithClassList("button", "fx-relay-menu-generate-alias-btn");
     generateAliasBtn.textContent = browser.i18n.getMessage("pageInputIconGenerateNewAlias");
 
+    // Create "Get unlimited aliases" button
+    const getUnlimitedAliasesBtn = createElementWithClassList("a", "fx-relay-menu-get-unlimited-aliases");
+    getUnlimitedAliasesBtn.textContent = browser.i18n.getMessage("popupGetUnlimitedAliases");
```

Comment:
We might want to avoid replacing the user's tab:

```suggestion
    getUnlimitedAliasesBtn.textContent = browser.i18n.getMessage("popupGetUnlimitedAliases");
    getUnlimitedAliasesBtn.setAttribute("target", "_blank");
    // See https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/
    getUnlimitedAliasesBtn.setAttribute("rel", "noopener noreferrer");
```